packages/beacon-node/src/api/rest/index.ts

```diff
@@ -21,7 +21,7 @@ export const beaconRestApiServerOpts: BeaconRestApiServerOpts = {
   port: 9596,
   cors: "*",
   // beacon -> validator API is trusted, and for large amounts of keys the payload is multi-MB
-  bodyLimit: 10 * 1024 * 1024, // 10MB
+  bodyLimit: 100 * 1024 * 1024, // 100MB for block + blobs
```

Comment:
100Mb seems bit excessive, shouldn't 20 be sufficient?

Reply:
incase blobs is increased later on or peerDAS but yea we can do that later as well so 20 might work for now, what do you think @wemeetagain ?

Reply:
yeah lets just do 20 for now